### ISSUE TITLE
Container deployment template or descriptor

### ISSUE THREAD

**Issue by oeymere** [crosbymichael/dockerui#130]
A feature that I think would really help is to have a way to provide a deployment description when creating a container.  When creating a container the UI pops up a dialog with a number of fields.  For my cases using docker I am creating new containers with the same parameters each time.  From the command line this is easy, I can reuse the same command with all the arguments and just change the image name.  It would be great in DockerUI if we could upload a file or create a template that we could apply from the create container dialog.  This would then apply all of the settings to the container when it runs.

**Comment by xianlubird** [comment]
I agree with you and I am working on it . But this repo seems that author no longer manger it .Many pull request are not merged and issues have nobody to answer. Why?

**Comment by kevana** [comment]
Maintainer here, I've been traveling without internet for the last 6 weeks and just got back yesterday. It'll take me a while to catch up with everything.

**Comment by kevana** [comment]
Related to #128. I like the idea

**Comment by blackbarn** [comment]
So...something like docker-compose support? Being able to specify a docker-compose.yml file (and perhaps a registry/list of them) to generate containers from? Would be awesome.....

**Comment by xianlubird** [comment]
It seems to be something different from docker compose, I think it's just add a label to a container like kubernets add labels to containers so we can select them easily

**Comment by deviantony** [comment]
Portainer has support for App templates, which are pre-defined container configurations that can be run in a few clicks. We also allow the users to specify their templates.

See https://github.com/portainer/portainer
